lxc/snapshot.go

```diff
@@ -100,6 +120,8 @@ func (c *cmdSnapshot) run(cmd *cobra.Command, args []string) error {
 
 	if c.flagNoExpiry {
 		req.ExpiresAt = &time.Time{}
+	} else if !stdinData.ExpiresAt.IsZero() {
```

Comment:
Whats the reason we can just assign stdinData.ExpiresAt to req.ExpiresAt before line 121?

Reply:
fixed